erpnext/www/book_appointment/index.py

```diff
@@ -14,7 +14,8 @@ def get_context(context):
 	if is_enabled:
 		return context
 	else:
-		frappe.local.flags.redirect_location = '/404'
+		frappe.redirect_to_message("Appointment Scheduling Disabled","Enable appointment booking from appointment booking settings"
+									,http_status_code=302,indicator_color="red")
```

Comment:
```suggestion
		frappe.redirect_to_message(_("Appointment Scheduling Disabled"), _("Appointment Scheduling has been disabled for this site")
									,http_status_code=302,indicator_color="red")
```

1. This message is meant for a user who is booking an appointment from the portal. They aren't aware that there is a doctype named **Appointment Booking Settings** in the system. This action is not expected from them so we just need to tell them that it's disabled.
2. Always make messages and title translatable.